bin/webpack-entries.js

```diff
@@ -130,6 +130,8 @@ const entries = {
 		'add-to-cart-form':
 			'./assets/js/atomic/blocks/product-elements/add-to-cart-form/index.tsx',
 		...getBlockEntries( '{index,block,frontend}.{t,j}s{,x}' ),
+		'wc-blocks-classic-template-revert-button-style':
+			'./assets/js/templates/revert-button/style.scss',
```

Comment:
I think it would be better to move the usage of the spread operator to be the last item in the object:

```suggestion
		'wc-blocks-classic-template-revert-button-style':
			'./assets/js/templates/revert-button/style.scss',
		...getBlockEntries( '{index,block,frontend}.{t,j}s{,x}' ),
```

Reply:
Thanks: I went ahead and added a comment to separate this new entry from the rest as the revert button is not really in the same category as the other blocks: it is meant specifically for usage within the template editor. [68e5415](https://github.com/woocommerce/woocommerce-blocks/pull/10388/commits/68e54150f47a7c2f82529102b6e9e3d17546f3f6)